cmd/optimizely/main.go

```diff
@@ -117,17 +118,7 @@ func loadConfig(v *viper.Viper) *config.AgentConfig {
 			}
 		}
 		if cache, ok := cmab["cache"].(map[string]interface{}); ok {
-			if cacheType, ok := cache["type"].(string); ok {
-				conf.CMAB.Cache.Type = cacheType
-			}
-			if cacheSize, ok := cache["size"].(float64); ok {
-				conf.CMAB.Cache.Size = int(cacheSize)
-			}
-			if cacheTTL, ok := cache["ttl"].(string); ok {
-				if duration, err := time.ParseDuration(cacheTTL); err == nil {
-					conf.CMAB.Cache.TTL = duration
-				}
-			}
+			conf.CMAB.Cache = cache
```

Comment:
We do not support partial conf with this? I mean cache config should have all fields, type, size, and ttl always?